go/vt/vtgate/planbuilder/builder.go

```diff
@@ -185,20 +187,20 @@ func buildDBDDLPlan(stmt sqlparser.Statement, vschema ContextVSchema) (engine.Pr
 			return engine.NewRowsPrimitive(make([][]sqltypes.Value, 0), make([]*querypb.Field, 0)), nil
 		}
 		if !ksExists {
-			return nil, vterrors.Errorf(vtrpcpb.Code_FAILED_PRECONDITION, "cannot drop database '%s'; database does not exists", ksName)
+			return nil, mysql.NewSQLError(mysql.ERDbDropExists, mysql.SSUnknownSQLState, "Can't drop database '%s'; database doesn't exists", ksName)
```

Comment:
nit: we can keep the error message as is. `Can't` `doesn't` does not seem good words to be in error message

Reply:
We took these messages from the mysql documentation https://dev.mysql.com/doc/mysql-errors/5.7/en/server-error-reference.html